inhouse/command_handlers/match.py

```diff
@@ -119,7 +120,7 @@ async def send_channel(self, member: discord.Member, channel: discord.VoiceChann
         except Exception as e:
             print(e)
             if member != None and channel != None:
```

Comment:
I've figured out why this doesn't show a lot of the time: Occasionally, `member` will be `None`, so the move above will throw an exception. This join message then never gets returned because we can't pass the condition here, and if we did we'd get another exception from the `member.id` on line 123. I'm really not entirely sure why it fails sometimes, seems to be pretty inconsistent on who it gets as `None`. I thought it had to do with the "Member" role but seems to do it sometimes even if someone has it. 

TL;DR, this chunk of code will basically never do anything. Not necessary to change in this PR, but I think we should change it to just send the channels and not @ the missing members in the short term.

Reply:
Interesting... will maybe fix on bug fix branch and open PR